guidelines/frontend-coding-standard/import-export.md

```diff
@@ -0,0 +1,61 @@
+# import / export
+
+## Prefer named import over default import
```

Comment:
better state the conclusion first in first section(as TLDR;), then provide our reason